timeslots/handler.go

```diff
@@ -11,14 +11,27 @@ type TimeslotData struct {
 	Time    []TimePeriod `json:"time"`
 }
 
+// time payload format that works
+// "time": [{"From": "2014-11-12T11:45:26.371Z", "To": "2017-11-12T11:45:26.371Z"}]
 func CreateTimeslots(w http.ResponseWriter, r *http.Request) {
```

Comment:
This might be a `PUT` operation for a period of time e.g. day, so it will remove old timeslots for the day and create new ones.

Reply:
in this case, also check that all timeslots belong to the specified day.